main.py

```diff
@@ -1,50 +1,73 @@
-gameboard={'topleft':' ','topmid':' ','topright':' ',
-           'midleft':' ','midmid':' ','midright':' ',
-           'bottomleft':' ','bottommid':' ','bottomright':' '}
-def showboard(board):
-    print('\n')
-    print(gameboard['topleft'],'|',gameboard['topmid'],'|',gameboard['topright'])
-    print('---------')
-    print(gameboard['midleft'],'|',gameboard['midmid'],'|',gameboard['midright'])
-    print('---------')
-    print(gameboard['bottomleft'],'|',gameboard['bottommid'],'|',gameboard['bottomright'])
-    print('\n')
-
-def win_check(gb,t):
-    return (
-        (gb['topleft']==t and gb['topmid']==t and gb['topright']==t) or
-        (gb['midleft']==t and gb['midmid']==t and gb['midright']==t) or
-        (gb['bottomleft']==t and gb['bottommid']==t and gb['bottomright']==t) or
-        (gb['topleft']==t and gb['midleft']==t and gb['bottomleft']==t) or
-        (gb['topmid']==t and gb['midmid']==t and gb['bottommid']==t) or
-        (gb['topright']==t and gb['midright']==t and gb['bottomright']==t) or
-        (gb['topleft']==t and gb['midmid']==t and gb['bottomright']==t) or
-        (gb['topright']==t and gb['midmid']==t and gb['bottomleft']==t)
-    )
-
-turn='X'
-check_list = []
-for i in range(9):
-    showboard(gameboard)
-    print('Make a move, player',turn)
-    move=input()
-    while True:
-        if move not in gameboard.keys():
-            print('Invalid position. Please enter again.')
-            move=input()
-        elif move in check_list:
-            print('That position is already taken. Please enter another.')
-            move=input()
-        else:
+'''
+This is a side project by Pranav Shridhar and Sandeep Pillai to implement ML to beat a basic game of
+tic tac toe, using re-inforced learning tactics.
+'''
+while(True):
+    gameboard = {'7':' ','8':' ','9':' ',
+                 '4':' ','5':' ','6':' ',   # This is how the basic board looks like, analogue to the numpad.
+                 '1':' ','2':' ','3':' '}
+
+    print("\n"*100, "Welcome to Tic-Tac Toe! Use your numpad to input the blocks!")
+
+
+    def showboard():       #function prints the board - commend Sandeep for beautification.
```

Comment:
Please avoid unecessary commenting.

Reply:
It's not unnecessary, we know the board looks like a numpad now. -.-

Reply:
`#function prints the board - commend Sandeep for beautification.`
_This_ doesn't seem particularly necessary and not the comment about numpad.